.github/workflows/lint.yml

```
@@ -11,7 +11,7 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

The reason why we are using ubuntu-18.04 for everything, is that we need it in order to generate binaries that work across Linux versions. So we use it everywhere for consistency.
```suggestion
    runs-on: ubuntu-18.04
```